app/src/test/java/com/example/trainingintive/viewmodels/MainViewModelTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.trainingintive.viewmodels
+
+import com.example.trainingintive.navigators.MainNavigator
+import com.example.trainingintive.util.MainScreenEvent
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Test
+
+class MainViewModelTest {
+
+    val navigator: MainNavigator = mockk(relaxUnitFun = true)
+    val tested = MainViewModel(navigator)
+
+    @Test
+    fun `send Logout event`() {
```

Comment:
This test name is unclear. You should briefly explain in test name – what should happen, when, in which case, etc. so for example something like "Should send Logout event when logout called" or "Should execute sendEvent on navigator with logout event when logout called".